packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersPage.tsx

```diff
@@ -32,9 +32,35 @@ export const InviteMembersPage = withCardStateProvider(() => {
       </ContentPage>
       <SuccessPage
         title={title}
-        // text={localizationKeys('userProfile.profilePage.successMessage')}
-        text={'Invitations successfully sent'}
+        content={<InvitationsSentMessage />}
+        Breadcrumbs={OrganizationProfileBreadcrumbs}
       />
     </Wizard>
   );
 });
+
+export const InvitationsSentMessage = () => {
+  return (
+    <Flex
+      direction='col'
+      center
+      gap={4}
+    >
+      <Flex
```

Comment:
We're using a similar icon/icon container in AvatarUploader. Maybe extract and reuse?